californium-core/src/main/java/org/eclipse/californium/core/observe/Observation.java

```diff
@@ -52,6 +52,20 @@ public Request getRequest() {
 	}
 
 	/**
+	 * @return a cloned observation with a shallow clone of request
+	 */
+	public Observation cloneObservation() {
```

Comment:
May be we can move it to a util class, because the "InMemoryObservationStore" is not the only one using that shallow clone?